rootfs/api/migrations/0019_auto_20160930_0645.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-09-30 06:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0018_auto_20160908_1748'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='build',
+            name='failed',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='config',
+            name='failed',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
```

Comment:
I never see the `failed` field for those used anywhere - Can you point to where? I figured only `Release` needed it since that's the one that dictates if things are okay or not, etc